### ISSUE TITLE
Should defaultContentType have a default?

### ISSUE THREAD

**Issue by fmvilas** [asyncapi/asyncapi#319]
[defaultContentType](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#default-content-type) doesn't have a default value and it's not listed on the AsyncAPI Object, even though it's at root level and must be there.

Should we make this field mandatory or have a default value of, for example, `application/json`?

Thanks, @jstoiko for [your work on the TCK](https://github.com/asyncapi/tck/pull/1), it helped raise the issue.

**Comment by derberg** [comment]
I don't think it should be mandatory, 
it for sure should have a default and best would be `application/json`
this is how I would expect default to work, not see him but have him, and only write explicitly if I want to change it

**Comment by jonaslagoni** [comment]
I agree with @derberg, like the idea of keeping the AsyncAPI file as simple as possible (default over mandatory) 👍

**Comment by fmvilas** [comment]
Yes, I agree too. Thanks, folks!

**Comment by jonaslagoni** [comment]
Is there any of you who wants to champion this? 🙂 Or can we consider this issue as `needs champion`? 🤔

**Comment by Anshul439** [comment]
Can I work on this issue?

**Comment by fmvilas** [comment]
Absolutely, @Anshul439. Go ahead 👍
